src/components/OrganizationUnitComboBox/OrganizationUnitComboBox.tsx

```diff
@@ -56,7 +59,6 @@ export const OrganizationUnitComboBox = ({
                         onFocus={suggestionsVisibility.setTrue}
                         onChange={(e) => {
                             setSelectedValue(null);
-                            onChange(null);
```

Comment:
not sure about that, it breaks synchronization between internal and external state